workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectCreation/GameObjectInitializationSystem.cs

```diff
@@ -40,6 +41,7 @@ private struct RemovedEntitiesData
         private ViewCommandBuffer viewCommandBuffer;
         private EntityGameObjectCreator entityGameObjectCreator;
         private EntityGameObjectLinker entityGameObjectLinker;
+        private EntityManager entityManager;
```

Comment:
What is this for? Also, can't we just access the EntityManager of a system directly like in line 52?